mar_13_assignment/gonza.md

```diff
@@ -1,3 +1,25 @@
-## This is my homework
-# testing MD
-**Bold**
+## SUPREME COURT OF THE UNITED STATES
+### IN THE SUPREME COURT OF THE UNITED STATES
+
+
+Place: Washington, D.C.
+
+Date: March 12, 2050
+<br></br><br></br>
+Mr. Chief Justice, and may it please the Court:
+
+Since the implementation of the "Grab Them By The Uterus" Act, almost 19 years ago. According to it, every woman in Puerto Rico has received their pregnancy prosecution status after each period, by email. Whenever their Smart Tampon detects flow, User Data was put to work. If her life style and behaviors would prove at least 95% chance of the unborn to commit any crimes during their lifetime, the woman would be sentenced for instigation to criminalize their future son. In this way, as stated by the Act, every woman would know their current status on a monthly basis. If the status was *guilty* and decided to get pregnant anyway, it would be "their choice to do so".
```

Comment:
Since the implementation of the "Grab Them By The Uterus" Act, almost 19 years ago, every woman in Puerto Rico received their pregnancy prosecution status after each period, by email.

If their status was *guilty* and they decided to get pregnant anyway, it would be "their choice to do so".